smk/binning_preparation.smk

```diff
@@ -280,6 +280,16 @@ def get_assemblies_for_scaf_type(wildcards):
 
     return(asms)
 
+def calculate_batch_filesize(wildcards):
+    filtered_batch_size = (max_job_ram_gb-2)/22*1024
+    if wildcards.scaf_type.endswith("nanopore"):
+        # assuming that size filtering doesn't drastically reduce file size
+        return(round(filtered_batch_size))
+    else:
+        # 2500bp filtering reduces file size by approx. half,
```

Comment:
Perhaps we can avoid this if-else if we directly measure the filesize after writing in the shadow folder. Please see below.